src/android/barcodescanner.gradle

```diff
@@ -17,4 +17,4 @@ android {
         exclude 'META-INF/NOTICE'
         exclude 'META-INF/LICENSE'
```

Comment:
It is missing bump the version of this plugin

Reply:
We haven't done the bump for original change either (https://github.com/feedhenry/phonegap-plugin-barcodescanner/commit/b1902a50db139584f4f2c81c6a140b9c2792ae89).
We are not going to publish this outside of GH, so perhaps that's why @davidffrench  didn't change the version.

btw: there already is 6.0.1 in upstream.

Reply:
This is exactly why. this should be exactly 6.0.0 with the hardcoded support library. I understand this should trigger a version bump but now we are very different from the upstream 6.0.1 so it is better to not update the version in my opinion.

Reply:
👍